docs/src/examples/knapsack.md

```diff
@@ -94,35 +92,21 @@ df = CSV.read("knapsack.csv", DataFrame)
 ```@example dwave-knapsack
 using JuMP
 using ToQUBO
-using Anneal # <- Your favourite Annealer / Sampler / Solver here
+using DWaveNeal # <- Your favourite Annealer / Sampler / Solver here
 
-# -> Model <-
-model = Model(() -> ToQUBO.Optimizer(SimulatedAnnealer.Optimizer))
+model = Model(() -> ToQUBO.Optimizer(DWaveNeal.Optimizer))
 
 n = size(df, 1)
 c = collect(Float64, df[!, :cost])
 w = collect(Float64, df[!, :weight])
 C = round(0.8 * sum(w))
 
-# -> Variables <-
-@variable(model, x[i=1:n], Bin)
-
-# -> Objective <-
+@variable(model, x[1:n], Bin)
 @objective(model, Max, c' * x)
-
-# -> Constraint <-
 @constraint(model, w' * x <= C)
 
-# ->-> Run! ->->
 optimize!(model)
 
 # Add Results as a new column
-insertcols!(
-   df,
-   3, 
-   :select => map(
-      (ξ) -> (ξ > 0.0) ? "Yes" : "No",
-      value.(x),
-   ),
-)
+insertcols!(df, 3, :select => map((ξ) -> (ξ > 0.0) ? "Yes" : "No", value.(x)))
```

Comment:
Mention how exciting is solving a problem with a quantum annealer maybe?

Reply:
Added to #63